tests/integration/test_tickets.py

```diff
@@ -51,35 +51,6 @@ def test_tickets_GET_one(client, test_database, auth_headers):
     assert response.json == {"message": "Ticket not found"}
 
 
-def test_tickets_POST(client, auth_headers):
```

Comment:
We shouldn't be deleting tests when adding a feature. Please explain why these deletions are necessary.

Reply:
It was replaced by the one added below